static/app/components/modals/debugFileCustomRepository/appStoreConnect/stepThree.tsx

```diff
@@ -14,6 +16,14 @@ type Props = {
 function StepThree({stepThreeData, onSetStepOneData}: Props) {
   return (
     <Fragment>
+      <Alert type="info">
+        {tct(
+          'Please enter the iTunes credentials that Sentry should use to download dSYMs from App Store Connect. It is recommended to [linkToDoc:create a new Apple ID] with the "Developer" role for this.',
+          {
+            docLink: <ExternalLink href="https://support.apple.com/en-us/HT204316" />,
```

Comment:
Is this link working?  in the screenshot on this PR it doesn't show underlined as the other link does

Reply:
good catch :)